wazuh/wazuh-kibana-app
# Change the method to make the redirect

### Description
Change the method to perform the redirection when clicking on the button to view the agent or view the agent configuration, so that the view is not broken when refreshing it. 
 
### Issues Resolved
- #5454

### Evidence

![image](https://github.com/wazuh/wazuh-kibana-app/assets/63758389/e68c90d2-e5ca-44ac-8508-f59f434e6469)

![image](https://github.com/wazuh/wazuh-kibana-app/assets/63758389/3995c35a-9c2f-438d-8a0a-3875ab514abc)

### Test

Navigate to an agent or the configuration of an agent using the table buttons, 
- Make sure that  `tab` and `agent` parameter appears in the url. 
- Refresh the page and the correct information should be displayed.
- In the agent view navigate to `Inventory data | Stats | Configuration` and refresh the page should reload the same information.

### Check List
